src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -89,6 +92,34 @@ private JObject GetInvokeResult(byte[] script, Signers signers = null)
             return json;
         }
 
+        private JObject ToJson(StackItem p, ref int max)
+        {
+            if (max <= 0) throw new InvalidOperationException();
+            max--;
+
+            if (p is InteropInterface result)
+            {
+                var iterator = result.GetInterface<IIterator>();
+                if (iterator != null)
+                {
+                    VM.Types.Array resultList = new();
+                    while (iterator.Next())
+                    {
+                        resultList.Add(iterator.Value());
+                        max--;
+                        if (max <= 0) throw new InvalidOperationException();
+                    }
+
+                    JObject ret = new();
+                    ret["type"] = "InteropInterface";
+                    ret["value"] = ToJson(resultList, ref max);
+                    return ret;
+                }
+            }
```

Comment:
At least should show some warning and return current `InteropInterface` value.
```suggestion
string warning = null;
            if (max <= 0) throw new InvalidOperationException();
            max--;

            if (p is InteropInterface result)
            {
                var iterator = result.GetInterface<IIterator>();
                if (iterator != null)
                {
                    VM.Types.Array resultList = new();
                    while (iterator.Next())
                    {
                        resultList.Add(iterator.Value());
                        max--;
                        if (max <= 0)
                        {
                            max++;
                            warning = "MaxIteratorItems is acheived. Can't show all values.";
                            break;
                        }
                    }

                    JObject ret = new();
                    ret["type"] = "InteropInterface";
                    ret["value"] = ToJson(resultList, ref max);
                    ret["warning"] = warning;
                    return ret;
                }
            }
```